src/routes/artist.js

```diff
@@ -0,0 +1,14 @@
+// src/routes/artist.js
+const express = require('express');
+const artistController = require('../controllers/artist');
+
+const router = express.Router();
+
+router.post('/', artistController.createArtist);
+router.get('/', artistController.getAllArtists); 
+router.get('/:id', artistController.getArtistById);
+router.put('/:id', artistController.putArtist); 
```

Comment:
Same as above with the naming, as you use ById in get the artist by id, I would stick with it for the put, update and delete methods as well.